SETUP_NANO_BANANA.md

```
@@ -0,0 +1,343 @@
+      </Tabs>
+    </div>
+  );
+}
+```
+
+## Pedir Ajuda ao Claude
+
+Você pode pedir ao Claude Code para:
+
+1. **Integrar a API**:
+   ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)